src/aap_eda/services/activation/restart_helper.py

```diff
@@ -23,25 +23,35 @@
 LOGGER = logging.getLogger(__name__)
 
 
-def system_restart_activation(activation_id: int, delay_seconds: int) -> None:
+def system_restart_activation(
+    process_parent_type: str, id: int, delay_seconds: int
```

Comment:
I have reservations about referring to a `process_parent_type`.  While I grasp that the intention seems to be to refer to X (where X is the thing to process) versus X-Instance (a specific processing) the processing of X via an X-Instance is incidental (architecture/design-wise, not in an auditable sense) to what we want to accomplish which is processing an X. 
I think the name should reflect this higher-level conceptual idea rather than implicitly exposing an aspect of the implementation.

Reply:
@jshimkus-rh Do you have a recommended name to replace `process_parent_type`? Here the `process` is a short for `rulebook_process`

Reply:
I think simply dropping `parent` from the name  would suffice.
This would also apply more broadly to every other use of `parent` in the change excepting anything that actually has a parent-child relation, of course.